src/index.js

```diff
@@ -3,7 +3,7 @@
  */
 
 import debugFactory from 'debug';
-import Limiter, { memoryAdapter, redisAdapter } from 'ratelimiter';
+import Limiter, { memoryAdapter, nullAdapter, redisAdapter } from 'ratelimiter';
```

Comment:
You don't have to define the `nullAdapter` in `ratelimiter` ? (ratelimiter isn't modified in the PR)